src/fmu/dataio/export/rms/_utils.py

```diff
@@ -10,7 +10,13 @@
 from fmu.dataio.exceptions import ValidationError
 from fmu.dataio.export import _enums
 
-from ._conditional_rms_imports import import_rms_package
+try:
+    import rmsapi
+except ImportError as e:
+    raise ImportError(
+        "Module 'rmsapi' is not available. You have to be inside "
+        "RMS to use this function."
+    ) from e
```

Comment:
Had to move the import directly into here to get mypy to accept rmsapi objects as typehints